run_action.py

```diff
@@ -495,6 +494,138 @@ def dismiss_change_requests(
         # Avoid triggering abuse detection
         time.sleep(10)
 
+    if auto_resolve_conversations:
+        resolve_conversations(
+            github_token=github_token,
+            repo=repo,
+            pull_request_id=pull_request_id,
+            github_api_timeout=github_api_timeout,
+        )
+
+
+def conversation_threads_to_close(repo, pr_number, github_token, github_api_timeout):
+    """Generator of unresolved conversation threads to close
+
+    Uses the GitHub GraphQL API to get conversation threads for the given PR.
+    Then filters for unresolved threads and those that have been created by the action.
```

Comment:
Question: Where do we select those created by the action?

Reply:
Fair point. I guess we select open conversations started by the github-action user: https://github.com/platisd/clang-tidy-pr-comments/pull/76/files#diff-9981508c066d4bcb876bb89f7092ac196ce743485c2782d4142e6eb5e7c34457R561-R563
which might select conversations not actually started by the action. There's no message prefix to check for those though, right? Do you have another idea how to make the selection tighter?

Reply:
Maybe it's a good idea to exploit that every message [starts](https://github.com/platisd/clang-tidy-pr-comments/blob/master/run_action.py#L196C22-L196C31) with `⚠️ .... ⚠️`?

Reply:
Ah, yes. Maybe pull up that string to either module level "constant" or pass it around like the warning prefix? Just so the two places cannot drift?

Reply:
done. Tried to be explicit with using the kw-arg version in existing code.

Reply:
Maybe it's enough with the `single_comment_prefix` for now, but what about a regex that checks that the first line starts and finishes with `:warning:`?

Reply:
Updated to use a regex